src/hotspot/share/runtime/thread.cpp

```diff
@@ -3148,8 +2947,19 @@ void JavaThread::print_on_error(outputStream* st, char *buf, int buflen) const {
   return;
 }
 
+
 // Verification
 
+void JavaThread::frames_do(void f(frame*, const RegisterMap* map)) {
```

Comment:
Where does this come from ??

Reply:
It was moved from code above that was in the middle of the stack overflow code.  I moved it to right before it's called.